docs/configuration.md

```diff
@@ -0,0 +1,114 @@
+# Configuration
+
+[More configuration information is available in the Neon documentation.](https://neongeckocom.github.io/neon-docs/quick_reference/configuration/)
+
+All configuration is done in the `neon.yaml` file. This file is located in the `/home/neon/xdg/neon/neon.yaml` directory by default. After making changes to the file, you must restart the Neon core services in Yacht for the changes to take effect.
+
+Neon core services:
+
+- [neon-gui](https://github.com/NeonGeckoCom/neon_gui)
+- [neon-enclosure](https://github.com/NeonGeckoCom/neon_enclosure)
+- [neon-audio](https://github.com/NeonGeckoCom/neon_audio)
+- [neon-skills](https://github.com/NeonGeckoCom/neoncore)
+- [neon-speech](https://github.com/NeonGeckoCom/neon_speech)
+- [neon-messagebus](https://github.com/NeonGeckoCom/neon_messagebus)
+
+## Common configuration options
+
+Common configurations are found in the `neon.yaml` file.
+
+!!! warning
+    YAML is a whitespace-sensitive language. Be sure to maintain proper indentation when editing the `neon.yaml` file.
+
+Specific configuration changes are also detailed in blog posts by community dev Mike Gray. [Read the blog posts](https://blog.graywind.org/tags/neon/).
+
+### Logging verbosity
+
+To increase or decrease the verbosity of the logs, change the `LOG_LEVEL` value. The default value is `INFO`.
+
+```yaml
+LOG_LEVEL: DEBUG
+```
+
+All valid Python logging levels are available. For reference, see the [Python logging documentation](https://docs.python.org/3/library/logging.html#logging-levels).
+
+### Units of measurement
+
+To switch between metric and imperial units, change the `system_unit` value. The default value is `imperial`.
+
+```yaml
+system_unit: metric
+```
+
+To switch between 12-hour and 24-hour time, change the `time_format` value. The default value is `half`, or 12-hour time.
+
+```yaml
+time_format: full
+```
+
+### Timezone
+
+To change the timezone manually, change the `location.timezone` value. The default value is `America/Los_Angeles`.
+
+```yaml
+location:
+  timezone: America/Chicago
+```
```

Comment:
This will affect the system defaults, but its worth noting that clients/nodes are generally expected to supply this information with each request and their configuration may be formatted differently